test/e2e/workload_clusters_test.go

```diff
@@ -20,15 +21,15 @@ func runWorkloadClusterFlow(test *framework.MulticlusterE2ETest) {
 	test.DeleteManagementCluster()
 }
 
-func TestVSphereKubernetes121WorkloadClusterDemo(t *testing.T) {
-	provider := framework.NewVSphere(t, framework.WithUbuntu120())
+func TestVSphereMulticlusterKubernetes121WorkloadClusterDemo(t *testing.T) {
```

Comment:
nit: Could you add the `Multicluster` at the end?
like `TestVSphereKubernetes121MulticlusterWorkloadClusterDemo`
I think most tests follow `Test[Provider]Kubernetes[Kube-version][Type of test/scenario]`

Reply:
Yes thanks for this info. I will update.